src/pep257.py

```diff
@@ -1592,6 +1607,18 @@ def check_no_signature(self, function, docstring):  # def context
             if function.name + '(' in first_line.replace(' ', ''):
                 return D402()
 
+    @check_for(Function)
+    def check_capitalized(self, function, docstring):
+        """D403: First word of the first line should be properly capitalized.
+
+        The [first line of a] docstring is a phrase ending in a period.
+
+        """
+        if docstring:
+            first_word = eval(docstring).split()[0]
```

Comment:
So @Nurdok is correct, what you get back looks like

```py
(3, '"""Biz baz."""', (3, 3), (3, 17), '   """Biz baz."""\n')
```

So you're doing

```py
eval('"""Biz baz."""\n')  # => 'Biz baz.'
```

But it's functionally equivalent to `ast.literal_eval` in this case. I would still prefer `ast.literal_eval` to be used here, but I'm not as stringently against using `eval` as I was earlier.

We should probably add a comment regardless to reassure future reviewers as well.